benchmarks/benchmark_sqzb.py

```diff
@@ -243,6 +276,25 @@ def main(args: argparse.Namespace):
                              "Otherwise, we use Poisson process to synthesize "
                              "the request arrival times.")
     parser.add_argument("--seed", type=int, default=0)
+    parser.add_argument("--guided-json", action="store_true",
+                        help="Enable guided json")
+    parser.add_argument("--json-template", type=str, default=None,
+                        help="Path to guided json template. "
+                             "If not given, default json template is applied.")
+    parser.add_argument("--enable-lora", action='store_true',
+                        help="Enable Multi-LoRA")
+    parser.add_argument("--random-lora", action='store_true',
+                        help="Shuffle lora-pattern randomly.")
+    parser.add_argument("--lora-pattern", type=parse_lora_pattern, default=[],
+                        help="Multi-batch LoRA ids. Skip LoRA for empty IDs. "
+                             "e.g.: ,,sql-lora,sql-lora-2"
+                             "LoRAs are applied in round-robin manner.")
 
     args = parser.parse_args()
+
+    if args.enable_lora:
+        assert args.lora_pattern, "LoRA pattern must be set."
+    if args.guided_json:
+        assert args.json_template, "Guided JSON template must be given."
```

Comment:
How about replace assert with raise ValueError for better error clarity when lora_pattern or json_template are missing.
Example:
```
if args.enable_lora and not args.lora_pattern:
    raise ValueError("LoRA pattern must be set when LoRA is enabled.")
if args.guided_json and not args.json_template:
    raise ValueError("Guided JSON template must be provided when guided JSON is enabled.")

```